gulpfile.js

```diff
@@ -2,28 +2,21 @@
 
 var gulp = require("gulp");
 var mocha = require("gulp-mocha");
-var jslint = require("gulp-jslint");
+var eslint = require("gulp-eslint");
 
 var libSources = ["gulpfile.js", "lib/**/*.js"];
 var testSources = ["test/**/*.js"];
 
 gulp.task("lint-lib", function () {
     return gulp.src(libSources)
-        .pipe(jslint({
-            node: true
-        }))
-        .pipe(jslint.reporter("stylish"));
+        .pipe(eslint())
+        .pipe(eslint.format());
 });
 
 gulp.task("lint-test", function () {
-    return gulp.src(testSources).pipe(jslint({
-        options: {
-            node: true,
-            this: true
-        },
-        globals: ["describe", "it", "before", "after", "beforeEach", "afterEach"],
-        noFail: true
-    }));
+    return gulp.src(testSources)
+        .pipe(eslint())
```

Comment:
Not really important, but the lint tasks could be combined now that there are no differences in the configuration.